src/MessagePack.Generator.Roslyn3/MessagePack.Generator.Roslyn3.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>11</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <DefineConstants>ROSLYN3</DefineConstants>
+    <AnalyzerLanguage>cs</AnalyzerLanguage>
+    
+    <!-- Mainly used for Unity, Unity 2021.3 has Roslyn 3.8.0 (https://docs.unity3d.com/Manual/roslyn-analyzers.html) -->
+    <MicrosoftCodeAnalysisVersion>3.9.0</MicrosoftCodeAnalysisVersion>
```

Comment:
@Y-YoL I got 3.9.0 from your branch, but [this unity doc](https://docs.unity3d.com/Manual/roslyn-analyzers.html) says that 3.8.0 must be used. Can you explain? Do we need to change this to 3.8?